File: app/src/main/java/com/example/roomapplication/roomdb/Note.java
package com.example.roomapplication.roomdb;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "note table")
public class Note {

    @PrimaryKey(autoGenerate = true)
    public int id;
    @ColumnInfo(name = "name")
    public String name;
    @ColumnInfo(name = "age")
    public String age;
    @ColumnInfo(name = "jobTitle")
    public String jobTitle;
    @ColumnInfo(name = "gender")
    public String gender;

    public Note(String name, String age, String jobTitle, String gender) {
        this.name = name;
        this.age = age;
        this.jobTitle = jobTitle;
        this.gender = gender;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAge() {
        return age;
    }

    public void setAge(String age) {
        this.age = age;
    }

    public String getJobTitle() {
        return jobTitle;
    }

    public void setJobTitle(String jobTitle) {
        this.jobTitle = jobTitle;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }
}

File: app/src/main/java/com/example/roomapplication/roomdb/Note.kt
package com.example.roomapplication.roomdb

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "note table")
class Note(@field:ColumnInfo(name = "name") var name: String, @field:ColumnInfo(name = "age") var age: String, @field:ColumnInfo(name = "jobTitle") var jobTitle: String, @field:ColumnInfo(name = "gender") var gender: String) {
    @JvmField
    @PrimaryKey(autoGenerate = true)
    var id = 0

}